README.md

```diff
@@ -1,10 +1,177 @@
+
+This repository is the source of the [Official SigLens documentation](https://siglens.github.io/siglens-docs). We appreciate your interest in contributing and welcome your participation in making this resource even better. If you have any questions, please reach out to us on [Slack](https://www.siglens.com/slack.html).
+
+# Table of Contents
+
+- [Introduction](#introduction)
+- [Getting Started](#getting-started)
+- [Documentation Workflow](#documentation-workflow)
+  - [Markdown structure](#markdown-structure)
+- [How to Contribute](#how-to-contribute)
+  - [Content Guidelines](#content-guidelines)
+  - [First-Time Contributors](#first-time-contributors)
+- [Community and Support](#community-and-support)
+- [License](#license)
+
+## Introduction
+
+SigLens is engineered for excellence in managing massive volumes of data, from terabytes to petabytes, without compromising on speed or efficiency. At the heart of SigLens is our proprietary microindexing technology, which ensures that micro indices are only 1/100th the size of traditional indices, resulting in smaller index sizes and faster ingestion speeds. Moreover, SigLens utilizes dynamic columnar compression to reduce on-disk storage size to often just 1% of the incoming volume. This innovation not only enhances query times but also substantially lowers storage and compute requirements.
+
+## Getting Started
+
+Before you start contributing to SigLens Documentation, here are a few things you should know:
+
+- **Markdown**: Our documentation is written in Markdown. Familiarity with Markdown syntax is helpful but not required.
+- **Git and GitHub**: Contributions are made via GitHub. A basic understanding of how to fork a repository, create branches, and submit pull requests is necessary.
+
+**SigLens Platform**: Having a basic understanding of what SigLens offers will help you write better documentation. We encourage you to try out SigLens using 
+
+- The [Siglens Playground](http://playground.sigscalr.io:5122/), or
+- Installing SigLens in your local machine
+
+### &emsp; <a href="https://siglens.github.io/siglens-docs/installation/git" target="_blank">Git</a> &emsp; | &emsp; <a href="https://siglens.github.io/siglens-docs/installation/docker" target="_blank">Docker</a> &emsp;| &emsp; <a href="https://siglens.github.io/siglens-docs/installation/helm" target="_blank">Helm</a>
+
+## Documentation Workflow
+
+SigLens documentation uses [Docusaurus](https://docusaurus.io/docs) for creating and managing [SigLens Documentation](https://siglens.github.io/siglens-docs) website.
+
+To host this documentation on your local machine, follow these steps.
+
+1. Clone the repository
+    ```
+    git clone https://github.com/<username>/siglens-docs.git
```

Comment:
You should either replace `<username>` with `siglens` or add a step before this where they fork the repo and then in this step they clone their fork